models/engine/file_storage.py

```diff
@@ -48,14 +48,27 @@ def save(self):
         with open(self.__file_path, 'w') as f:
             json.dump(json_objects, f)
 
+    def get(self, cls, id):
+        """ A method to retrieve one object """
+        for object in self.__objects.values():
+            if object.__class__ == cls and object.id == id:
+                return object
+            return None
+
+    def count(self, cls=None):
+        """ A method to count the number of objects in storage """
+        if cls is None:
+            return len(self.all())
+        return len(self.all(cls))
```

Comment:
What if don't exist the class?

Reply:
If the class doesn't exist the second return would tell us. Because it won't be able to find it